### PR TITLE
Set up a 'Hello world' project with two apps: Backend

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS MILESTONE I:
1. Created a new Rails API app called 'hello-rails-back-end'.
2. Initialized your project with Git.
3. Made sure that my project has a Postgres database set up.
4. Created a table for storing my messages and created 5 different greetings.
5. Created an API endpoint that selects a random greeting from my table.

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "example.com"
+
+    resource "*",
+      headers: :any,
+      methods: [:get, :post, :put, :patch, :delete, :options, :head]
+  end
+end
```
Comment (AbelG101) [inline config/initializers/cors.rb]: - [ ] Great job incorporating the Rack CORS Middleware in your backend API! However, there appears to be a small configuration oversight on line 10 where you specify the origins. To resolve this, replace the placeholder example.com with the actual domain you intend to allow, such as `"http://localhost:3000"`. Alternatively, you can use the wildcard selector `*` to allow requests from any domain. Regardless of the option you choose, ensure that the error shown below is eliminated and requests are processed successfully without any CORS-related issues.
  277707843-c1aa607c-b618-4e15-a7cc-7a1c5adb2ea6
- Please refer to the [Rack CORS Middleware](https://github.com/cyu/rack-cors#rack-cors-middleware-) documentation for further guidance.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Kgomotso196 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- API is flawlessly Implemented✔
- Proper links provided for frontend and backend ✔
- Correct use of Git Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/initializers/cors.rb (hunk 1 above)
Comment (Kgomotso196) [reply config/initializers/cors.rb]: Thank you for your constructive feedback @AbelG101, working on it.

**Review by gemmen29 — APPROVED** [review]
Hi @Kgomotso196, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**